xtext-website/_posts/releasenotes/2020-03-03-version-2-21-0.md

```diff
@@ -10,6 +10,12 @@ Xtext 2.21.0 is mainly a maintenance release.
 
 ## Eclipse Integration
 
+## xtext-maven-plugin
+
+### testGenerate Goal
+
+The maven plugin has now two goals, generate and testGenerate. The new testGenerate goal runs during the generate-test-sources phase and uses the test class path of the maven project.
```

Comment:
maven => Maven
class path => classpath
Please make the goal names `monotype`
`generate`, `testGenerate`, `generate-test-sources`

Reply:
I changed this.